attestation-service/verifier/src/snp/mod.rs

```diff
@@ -258,6 +258,11 @@ mod tests {
     use super::*;
     use openssl::nid::Nid;
 
+    const VCEK: &[u8; 1360] = include_bytes!("../../test_data/snp/test-vcek.der");
```

Comment:
Might not related to the PR. But if we can have an agreement that the binary values should be all showed as hex or base64? see Line 250 in this file (I cannot add comment to line out of this PR)

Reply:
Yeah let's take this in a separate issue but I think it is probably a good idea.

Reply:
Ok. See https://github.com/confidential-containers/trustee/issues/372. And I think other fields of SNP should also do the same, such as https://github.com/confidential-containers/trustee/blob/main/attestation-service/verifier/src/snp/mod.rs#L240